documentation/existing.md

```diff
@@ -2,16 +2,22 @@
 title: Configurations on Existing Kubernetes Cluster
 ---
 
-If a Kubernetes cluster has not been deployed using Kubernetes-Anywhere, follow the instructions below to enable the vSphere Cloud Provider. These steps are not needed when using Kubernetes-Anywhere, they will be done as part of the deployment.
+# Enabling vSphere cloud provider
+## Preferred method
+If user has deployed kubernetes cluster on vSphere then use automated way of enabling vSphere cloud provider. For more details, please visit [https://github.com/vmware/kubernetes/blob/enable-vcp-uxi/README.md](https://github.com/vmware/kubernetes/blob/enable-vcp-uxi/README.md). If pre-requisites are not applicable, then follow below manual steps.
 
-All the steps mentioned below have been automated. For more details, please visit [https://github.com/vmware/kubernetes/blob/enable-vcp-uxi/README.md](https://github.com/vmware/kubernetes/blob/enable-vcp-uxi/README.md). If pre-requisites are not applicable, then follow below steps manually.
+Note: If kubernetes cluster is spanning across multiple vCeneters then please use below manual steps.
 
+## Manual steps (if needed) to enable vSphere cloud provider
+### Step - 1: Create a VM folder for Node VMs 
 
-## Create a VM folder for Node VMs
+Note: This is required only if kubernetes version is 1.8.x or below. Not required for kubernetes version 1.9.x
```

Comment:
Let's use the capital letter consistently for the term "Kubernetes". Same for other places.

Reply:
Done